#id number
int32 id

#time created
time time

#pose
geometry_msgs/Pose pose

#robot id assigned
int32 robotID

# 0 if completed, 1 if available, 2 if in progress
int32 status
